src/mapping_igvc/package.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>mapping_igvc</name>
```

Comment:
Would it be worth the time to make the messages their own package? 
I'm assuming this current one will contain everything regarding mapping (obstacle manager, occ grid publisher, etc.)

Reply:
I definitely considered this, and my initial setup for these message types was to put them in a separate package, just because the build stuff was a little easier. The balance here is that it logically makes sense for them to be in this package (I think), and reduces the number of packages in `src` (which hopefully keeps things a bit more accessible for new members), but does add a little bit of complexity to the build process (we need to make sure messages are built before executables) and breaks a ROS pseudo-standard (that messages have their own package). It won't take long to put these in a separate package, so if you feel at all strongly about this, reply as such and I can.